### ISSUE TITLE
Can’t connect Gen 1 lights

### ISSUE THREAD

**Issue by saiajin** [nschum/homebridge-twinkly#15]
Hi I have been trying to connect my Gen 1 lights to no avail... they do not auto discover.

**Comment by ortoberi** [comment]
I have first generation Twinkly lights and it works great. For me, the following is written in my settings:

{
            "name": "Twinkly",
            "ip": "192.168.88.100",
            "allowBrightnessControl": true,
            "accessory": "Twinkly"
},

BR,
David

**Comment by saiajin** [comment]
Thanks for your reply! I duplicated your entry:

  {
            "name": "TWINKLY1",
            "ip": "192.168.254.132",
            "allowBrightnessControl": true,
            "accessory": "Twinkly"
        }

This is what shows in logs:

3/6/2021, 8:25:16 AM [3/6/2021, 9:25:16 AM] Registering accessory 'homebridge-twinkly.Twinkly'
3/6/2021, 8:25:16 AM [3/6/2021, 9:25:16 AM] Registering platform 'homebridge-twinkly.Twinkly'

BUt the device still doesn't show up... I have no idea what I'm doing wrong.

**Comment by nschum** [comment]
@saiajin This is a networking problem. Your Twinkly didn't get a proper IP address from your router (via DHCP). That means it's unreachable from the device running Homebridge.

**Comment by saiajin** [comment]
I'm sure of the solution as it shows the same IP address in the Twinkly app and I have it a static ip.![image](https://user-images.githubusercontent.com/71667075/110220849-5ced4f00-7e7d-11eb-9df8-e2989955170b.jpeg)

**Comment by saiajin** [comment]
@nschum what steps can I take to troubleshoot?

**Comment by saiajin** [comment]
Here is the device connected to my router:

![image](https://user-images.githubusercontent.com/71667075/110251683-7bfbe780-7f36-11eb-8b5f-873b307c6dfe.jpeg)

I have my RPI connected via Wi-Fi and Ethernet at the same time could I hat cause the issue?

**Comment by saiajin** [comment]
@nschum I have my RPI connected via Wi-Fi and Ethernet at the same time could I that possibly cause the issue?

**Comment by nschum** [comment]
@saiajin You could try a ping the IP address from the device running homebridge and check the homebridge logs for errors.

**Comment by saiajin** [comment]
Hi what command would I type in Terminal to do this? I'm on Hoobs. Sorry I'm not super savvy but I follow direction pretty well.

**Comment by saiajin** [comment]
I figured it out:

hoobs@hoobs:~ $ ping 192.168.254.132
PING 192.168.254.132 (192.168.254.132) 56(84) bytes of data.
64 bytes from 192.168.254.132: icmp_seq=1 ttl=128 time=12.2 ms
64 bytes from 192.168.254.132: icmp_seq=2 ttl=128 time=9.65 ms
64 bytes from 192.168.254.132: icmp_seq=3 ttl=128 time=8.78 ms
64 bytes from 192.168.254.132: icmp_seq=4 ttl=128 time=15.6 ms
64 bytes from 192.168.254.132: icmp_seq=5 ttl=128 time=12.6 ms
64 bytes from 192.168.254.132: icmp_seq=6 ttl=128 time=14.7 ms
64 bytes from 192.168.254.132: icmp_seq=7 ttl=128 time=8.91 ms
64 bytes from 192.168.254.132: icmp_seq=8 ttl=128 time=7.79 ms

**Comment by saiajin** [comment]
This is my current config:
    {
            "name": "TWINKLY1",
            "platform": "Twinkly",
            "timeout": 5000,
            "ip": "192.168.254.132",
            "accessory": "Twinkly",
            "removeUnreachableDeviceMinutes": 0,
            "allowBrightnessControl": true
        },

**Comment by saiajin** [comment]
Any clues? as U can see Pings seem to work.....

**Comment by wtdeane** [comment]
I'm having the same problem after I had to reinstall my Hoobs box and all my plugins.  I hope this explanation sheds some light for you.

When I was on the previous version and having problems with my V1 lights breaking, the latest update fixed them.  Saiajin, installing fresh is having problems like I am.  Something is happening where previously installed versions are functioning properly, but if you are working with a new instal, not a deleted and reinstall, but a CLEAN instal, it remains broke.  Any help would be GREATLY appreciated.

Cheers!

**Comment by saiajin** [comment]
Yes it appears if you had them previously and updated they still work for them... but new installs can't get them to work... I bought a set to test out and the Homekit functionality was the main reason... but I can't seem to get any help.

**Comment by wtdeane** [comment]
Have you had any luck getting it to run on your gen1 lights?

**Comment by saiajin** [comment]
> Have you had any luck getting it to run on your gen1 lights?

Nope I gave up.. No response from Dev..... Still watching this though...

**Comment by nschum** [comment]
@saiajin Since you have access to the command line, could you try the bundled `twinkly` command?
`./bin/twinkly -v --discover`
`./bin/twinkly -v --mode movie 192.168.254.132`
